docs/explanation/fde-osd.rst

```diff
@@ -26,6 +26,16 @@ Limitations
 **Warning:** It is important to note that MicroCeph FDE *only* encompasses OSDs. Other data, such as state information for monitors, logs, configuration etc., will *not* be encrypted by this mechanism.
 
 
+Prerequisites
```

Comment:
Minor Nit: I think the pre-requisites section should be moved up maybe just after the overview.

Reply:
Ack, reasonable